motors/motor_driver/motor_driver.ino

```diff
@@ -0,0 +1,246 @@
+//if O is pressed, open the door
+//if C is pressed, close the door
+//if Q is pressed, recalibrate positioning of open vs closed
+//if K is pressed, enter custom mode
+
+
+#include <AccelStepper.h>
+#include <Wire.h>
+
+// Define a stepper motor for arduino
+AccelStepper stepper(1, 12, 11); // pin 12 = step/pulse, pin 11 = direction
+/*
+Note: AccelStepper mystepper(1, pinStep, pinDirection); A stepper motor controlled by a dedicated driver board. In this case a DM542T
+Source: https://www.pjrc.com/teensy/td_libs_AccelStepper.html
+
+As a result pin 10 = enable is not used
+*/
+
+#define LEFT -2308
+#define RIGHT 2308
+#define PRESSED 1
+
+enum Procedure {
+    calibrate       = 2,
+    setDoorState    = 3,
+    getDoorState    = 4,
+    numMessages
+};
+
+enum DoorState {
+    unknown         = 0,
+    transit         = 3,
+    close           = 99,
+    open            = 111,
+    manualCalibrate = 2
+} doorState;
+
+enum Position {
+    doorIsOpen      = 2 * RIGHT,
+    doorIsClosed    = 0,
+    doorInTransit
+} doorPosition;
+
+enum Priority {
+    priorityLow     = 0,
+    priorityHigh    = 1
+};
+
+typedef struct Header_t {
+    byte action;
+    byte procedure;
+    byte priority;
+};
+
+typedef struct Calibrate_t {
+  Header_t header;
+};
+
+typedef struct SetDoorState_t {
+  Header_t header;
+  byte targetState;
+};
+
+typedef struct GetDoorState_t {
+    Header_t header;
+    byte doorState;
+    short angle;
+};
+
+//typedef union I2CMessage_t {
+//    Calibrate_t calibrate;
+//    SetDoorState_t setDoorState;
+//    GetDoorState_t getDoorState;
+//};
+
+//volatile I2CMessage_t* messages[numMessages] = {0};
+//volatile byte msgIndex = 0;
+
+int speed = 500;            //initial speed of the motor
+int RightRotation = RIGHT;  //initial 90 degree rotation
```

Comment:
RightRotation and LeftRotation aren't necessary - they are the same as the #define of lines 19/20. Remove one or the other for clarity.